build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 group 'uk.gov.hmcts.reform'
-version '1.0.0'
+version '1.0.1'
```

Comment:
is this a bug fix?

Reply:
Yes, in production there was a scenario when health check was showing up but the actual requests were not being served.

Reply:
could you please more details about this what error did the service returned ?

Reply:
you are testing the whole functionality as part of health check which doesn't seem to be ideal.

Reply:
Please have a look at the story : [ROC-3551](https://tools.hmcts.net/jira/browse/ROC-3551)